DuckDuckGo/FavoritesOverlay.swift

```diff
@@ -79,9 +80,31 @@ class FavoritesOverlay: UIViewController {
         } else {
             layout.minimumInteritemSpacing = 10
         }
-        
-        collectionView.frame = view.bounds
         collectionView.reloadData()
+        
+    }
+    
+    private func setupCollectionView() {
+        collectionView = UICollectionView(frame: view.bounds, collectionViewLayout: layout)
+        
+        collectionView.register(UINib(nibName: "FavoriteHomeCell", bundle: nil), forCellWithReuseIdentifier: "favorite")
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.backgroundColor = .clear
+        view.addSubview(collectionView)
+        
+        collectionView.dropDelegate = self
+        collectionView.dragDelegate = self
+        collectionViewReorderingGesture.delegate = self
+        collectionView.addGestureRecognizer(collectionViewReorderingGesture)
+
+        // Size constraints
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
+        let widthConstraint = collectionView.widthAnchor.constraint(lessThanOrEqualToConstant: Constants.collectionViewMaxWidth)
```

Comment:
@afterxleep Instead of using a hard coded value, perhaps it would be better to use the width of the view (maybe minus margins) for the width anchor constant to better support all sizes of devices?

Reply:
@amddg44  This was my first approach but I wanted the view to look the same as the HomeView controller favorites for consistency.   I could change both views to be the same way, but not sure that'd be part of the scope.